src/nanoemoji/write_glyphmap_for_glyph_svgs.py

```diff
@@ -35,17 +36,30 @@ def main(argv):
     glyph_order = ttLib.TTFont(source_font).getGlyphOrder()
 
     input_files = sorted(
-        (Path(f) for f in input_files if f != source_font), key=lambda f: int(f.stem)
+        (Path(f) for f in input_files if f != source_font),
+        key=lambda f: f.stem,
+        reverse=True,
     )
 
     with util.file_printer(FLAGS.output_file) as print:
-        for input_file in input_files:
+        while input_files:
+            svg_file = input_files.pop()
+            assert svg_file.suffix in {".png", ".svg"}, f"What is {svg_file}"
+            bitmap_file = None
+            if svg_file.suffix == ".png":
+                bitmap_file = svg_file
+                svg_file = input_files.pop()
```

Comment:
you're relying on the fact that .svg and .png are listed one after the other, and that .png comes before .svg in the reversed list of arguments (because "p" is before "s" in alphabetical order). How about you only pass either the .svg or the .png instead of both, since we are only interested in the file.stem anyway.

Reply:
oh actually seems like you are using both the svg_file and bitmap_file, not just their stem..

Reply:
nevermind, the files are sorted like that because we sorted them immediately before, all good